src/app/mock-data.ts

```diff
@@ -1,4 +1,4 @@
-export const context = '/39156-5';
+export const linkId = '/39156-5';
```

Comment:
In this case, "linkId" does not provide enough meaning, since there many linkIds in a Questionnaire.  How about "phq9ScoreLinkId"?

Reply:
Fixed